apps/src/templates/studioHomepages/CourseBlocksTools.jsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import ContentContainer from '../ContentContainer';
+import ResourceCard from './ResourceCard';
+import styleConstants from '../../styleConstants';
+import i18n from "@cdo/locale";
+
+const pegasusContentWidth = styleConstants['content-width'];
+
+const styles = {
+  container: {
+    width: pegasusContentWidth,
+    display: "flex",
+    justifyContent: "space-between"
+  },
+  regularRow: {
+    marginBottom: 20
+  }
+};
+
+const CourseBlocksTools = React.createClass({
+  propTypes: {
+    isEnglish: React.PropTypes.bool.isRequired,
+    isRtl: React.PropTypes.bool.isRequired,
+    codeOrgUrlPrefix: React.PropTypes.string.isRequired
+  },
+
+  render() {
+    const headingText = this.props.isEnglish
+      ? i18n.courseBlocksToolsTitleTeacher()
+      : i18n.courseBlocksToolsTitleNonEn();
+
+    const cards = [
+      {
+        heading: i18n.courseBlocksToolsAppLab(),
+        description: i18n.courseBlocksToolsAppLabDescription(),
+        path: 'applab'
+      },
+      {
+        heading: i18n.courseBlocksToolsGameLab(),
+        description: i18n.courseBlocksToolsGameLabDescription(),
+        path: 'gamelab'
+      },
+      {
+        heading: i18n.courseBlocksToolsWebLab(),
+        description: i18n.courseBlocksToolsWebLabDescription(),
+        path: 'weblab'
+      },
+      {
+        heading: i18n.courseBlocksToolsWidgets(),
+        description: i18n.courseBlocksToolsWidgetsDescription(),
+        path: 'widgets'
+      },
+      {
+        heading: i18n.courseBlocksToolsInspire(),
+        description: i18n.courseBlocksToolsInspireDescription(),
+        path: 'inspire'
+      },
+      {
+        heading: i18n.courseBlocksToolsVideo(),
+        description: i18n.courseBlocksToolsVideoDescription(),
+        path: 'videos'
+      },
+    ];
+
+    return (
+      <ContentContainer
+        heading={headingText}
+        description={i18n.standaloneToolsDescription()}
+        isRtl={this.props.isRtl}
+      >
+        {[0,3].map(
```

Comment:
This doesn't seem particularly easy to read/understand. I think what you're wanting to do is just to group these into sets of 3 each (i.e. 3 per row)?

If so, it might be clearer to use `_.chunk` and do something like:
```js
_.chunk(cards.slice(0, 6), 3).map((group, rowIndex) => (
  <div>
    {group.map((card, cardIndex) => (
      <ResourceIndex/>
    )}}
   </div>
))
```
Might not end up looking all that different but I think it might be a little easier to understand. Up to you.